### PR TITLE
[Aftershock] Greeting Card snippets

### PR DESCRIPTION
**Pull request by Maleclypse**
#### Summary
Mods "Greeting Card Snippets for Aftershock"

#### Purpose of change
Adds lore, hooks, and depth to a world.

#### Describe the solution
Add's greeting cards with some snippets.  Part of the Aftershock Lore Project. 

#### Describe alternatives you've considered
None

#### Testing
TBD need to add spawns.

### CODE REVIEW

File: data/mods/Aftershock/snippets/personal_cards.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+[
+  {
+    "type": "snippet",
+    "category": "afs_personal_card",
+    "text": [
+      {
+        "id": "afs_card_1",
+        "text": "There is a picture of a shuttle taking off with happy people waving from the viewports.  \"Salus is cold. \n\n Our love is true \n\n I hope to see you \n\n On Croatan 2 \n\n  My dearest, please join me.  I don't believe there are many shuttles left before people will be stuck here forever.\""
```
Comment (Maleclypse) [inline data/mods/Aftershock/snippets/personal_cards.json:8]:
```suggestion
        "text": "There is a picture of a shuttle taking off with happy people waving from the viewports.  \"Salus is cold.\n\n Our love is true\n\n I hope to see you\n\n On Croatan 2\n\n My dearest, please join me.  I don't believe there are many shuttles left before people will be stuck here forever.\""
```

File: data/mods/Aftershock/snippets/personal_cards.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      },
+      {
+        "id": "afs_card_2",
+        "text": "A person wrapped up in bandages in a hospital bed is pictured on the front of this card.  \"Get well soon!!! \n\n Torrance, I'm sorry you feel ill but many new colonists forget to process any foodstuffs from Salus IV to remove the macronutrient.\""
```
Comment (Maleclypse) [inline data/mods/Aftershock/snippets/personal_cards.json:12]:
```suggestion
        "text": "A person wrapped up in bandages in a hospital bed is pictured on the front of this card.  \"Get well soon!!!\n\n Torrance, I'm sorry you feel ill but many new colonists forget to process any foodstuffs from Salus IV to remove the macronutrient.\""
```

File: data/mods/Aftershock/snippets/personal_cards.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        "id": "afs_card_3",
+        "text": "A picture of a bloody steak.  This card appears particularly old.  \"Gauchos Steaks.  Deliveries made monthly from Brazil to Salus IV.  Sign up today!\""
+      },
+      {
+        "id": "afs_card_4",
+        "text": "A glossy image of flowers and funeral statuary.  \"I'm sorry for your loss.  Ebby was an excellent grandparent and we hate for you that he was infected with the Moxphoria and wandered into the wastes.\""
+      },
+      {
+        "id": "afs_card_5",
+        "text": "An orbital view of Salus IV.  \"Happy Colonization Day!  \n\n Leonid, \n\n This place is cold as Hell but we've made a life out here in the last ten years.\""
```
Comment (Maleclypse) [inline data/mods/Aftershock/snippets/personal_cards.json:24]:
```suggestion
        "text": "An orbital view of Salus IV.  \"Happy Colonization Day! \n\n Leonid,\n\n This place is cold as Hell but we've made a life out here in the last ten years.\""
```

File: data/mods/Aftershock/snippets/personal_cards.json [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+        "id": "afs_card_3",
+        "text": "A picture of a bloody steak.  This card appears particularly old.  \"Gauchos Steaks.  Deliveries made monthly from Brazil to Salus IV.  Sign up today!\""
+      },
+      {
+        "id": "afs_card_4",
+        "text": "A glossy image of flowers and funeral statuary.  \"I'm sorry for your loss.  Ebby was an excellent grandparent and we hate for you that he was infected with the Moxphoria and wandered into the wastes.\""
+      },
+      {
+        "id": "afs_card_5",
+        "text": "An orbital view of Salus IV.  \"Happy Colonization Day! \n\n Leonid,\n\n This place is cold as Hell but we've made a life out here in the last ten years.\""
```
Comment (Maleclypse) [inline data/mods/Aftershock/snippets/personal_cards.json:24]:
```suggestion
        "text": "An orbital view of Salus IV.  \"Happy Colonization Day!\n\n Leonid,\n\n This place is cold as Hell but we've made a life out here in the last ten years.\""
```

File: data/mods/Aftershock/snippets/personal_cards.json [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+      },
+      {
+        "id": "afs_card_6",
+        "text": "An old style movie theater.  \"Showing the vintage film Aelita!  \n\n Tovarishch, \n\n Come with me to the movies.  My grandfather says that this movie is one of the classics.\""
```
Comment (Maleclypse) [inline data/mods/Aftershock/snippets/personal_cards.json:28]:
```suggestion
        "text": "An old style movie theater.  \"Showing the vintage film Aelita!\n\n Tovarishch,\n\n Come with me to the movies.  My grandfather says that this movie is one of the classics.\""
```

File: data/mods/Aftershock/items/lore_items.json [hunk 6]
```diff
@@ -0,0 +1,17 @@
+[
+  {
+    "type": "GENERIC",
+    "id": "afs_valentine_card",
```
Comment (actual-nh) [inline data/mods/Aftershock/items/lore_items.json:4]:
```suggestion
    "id": "afs_personal_card",
```

File: data/mods/Aftershock/items/lore_items.json [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+    "category": "books",
+    "symbol": ",",
+    "color": "red",
+    "name": { "str": "personal message card" },
+    "snippet_category": "afs_valentine",
```
Comment (actual-nh) [inline data/mods/Aftershock/items/lore_items.json:9]:
```suggestion
    "snippet_category": "afs_personal_card",
```

**Review by actual-nh — COMMENTED** [review]
A few things look like they missed getting changed when copying over from vanilla valentine's cards.

**Pull request merged**